src/content/components/IntervalMarkerOverview.js

```diff
@@ -154,8 +167,17 @@ class IntervalMarkerOverview extends PureComponent {
                 onMouseDown={this._onMouseDown}
                 onMouseMove={this._onMouseMove}
                 onMouseUp={this._onMouseUp}
-                onMouseOut={this._onMouseOut}
-                title={title}/>
+                onMouseOut={this._onMouseOut}/>
+        {
+          tooltipContents
```

Comment:
I'm fine doing it in a separate patch, please file an issue if you'd prefer this :)